api/controllers/load_balancer_test.go

```diff
@@ -162,3 +162,47 @@ func TestListLoadBalancers(t *testing.T) {
 	assert.Equal(t, 200, recorder.Code)
 	assert.Equal(t, loadBalancerSummaries, response)
 }
+
+func TestUpdateLoadBalancer(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockLoadBalancerProvider := mock_provider.NewMockLoadBalancerProvider(ctrl)
+	mockJobScheduler := mock_job.NewMockScheduler(ctrl)
+	controller := NewLoadBalancerController(mockLoadBalancerProvider, mockJobScheduler)
+
+	req := models.UpdateLoadBalancerRequest{
+		LoadBalancerID: "lb1",
+		Ports:          &[]models.Port{},
+		HealthCheck: models.HealthCheck{
+			Target:             "80",
+			Interval:           60,
+			Timeout:            60,
+			HealthyThreshold:   3,
+			UnhealthyThreshold: 3,
+		},
+	}
+
```

Comment:
Please target the `job_runner` branch, as it is the most up-to-date

Reply:
:+1: